test/widget/src/server/linked_in_auth_code_widget_test.dart

```diff
@@ -115,4 +115,37 @@ void main() {
 
     expect(find.text('AppBar title'), findsOneWidget);
   });
+
+  testWidgets('has PreferredSizeWidget type, if App Bar is provided', (
+    WidgetTester tester,
+  ) async {
+    final testWidget = testbed.simpleWrap(
+      child: linkedInAuthCodeWidget(
+        appBar: AppBar(),
+      ),
+    );
+
+    await tester.pumpWidget(testWidget);
+    await tester.pumpAndSettle();
+
+    expect(
+      find.byWidgetPredicate((widget) => widget is PreferredSizeWidget),
```

Comment:
please use finding by type as there is no callback that is not needed, but also its more readable